studio/components/interfaces/Authentication/Policies/PolicyEditor/PolicyRoles.tsx

```diff
@@ -0,0 +1,44 @@
+import { FC } from 'react'
+import { PostgresRole } from '@supabase/postgres-meta'
+
+import MultiSelect from 'components/ui/MultiSelect'
+
+interface Props {
+  roles: PostgresRole[]
+  selectedRoles: string[]
+  onUpdateSelectedRoles: (roles: string[]) => void
+}
+
+const PolicyRoles: FC<Props> = ({ roles, selectedRoles, onUpdateSelectedRoles }) => {
+  // @ts-ignore
+  const formattedRoles = roles.map((role) => {
+    return {
+      id: role.id.toString(),
+      name: role.name,
+      value: role.name,
+      disabled: false,
+    }
+  })
```

Comment:
does it mean:
- MultiSelect can only work with string id?
- `disabled: false` ?

Reply:
actually good point - should be fine with number too
realised that disabled is also unnecessary here - should be an optional field that only is true if we want it to be